user/pages/03.commerce2/02.developer-guide/04.customers/02.addresses/02.countries-and-subdivisions/docs.md

```diff
@@ -0,0 +1,175 @@
+
+![Admin ui for default country](../../images/address-countries-1.png)
+
+### Country subdivisions
+A country can have several levels of subdivisions that are used for addressing. In the United States that would be the state. In Brazil it would be the state and the municipality. In China it would be the province, the prefecture-level city, and the county. The *Address* module provides user-friendly address forms that provide dropdowns for these subdivisions, thus speeding up the data entry process and reducing mistakes.
+
+Subdivision data is provided by the Commerce Guys addressing library. Subdivisions are hierarchcial, with up to 3 levels: Administrative area -> Locality -> Dependent locality. For each level, there is a set list of options that is used to populate form select lists (dropdown menus). For example, both Australia and the United States have a single level of subdivisions, called *States*. If *United States* is selected as the country, then its 50 states and additional territories appear as *State* options. If *Australia* is selected, then its 8 state/territory options are displayed:
+
+![Australian subdivisions](../../images/address-countries-2.png)
+
+South Korea is an example of a country with an additional level of subdivisions. Once an administrative area, labeled *Do si*, is selected, a list of locality (City) options appears:
+
+![South Korean subdivisions](../../images/address-countries-3.png)
+
+The dataset is stored locally in JSON format. To view the actual data, see `commerceguys/addressing/resources/subdivision`.
+
+
+#### Why is country X missing subdivisions?
+The Commerce Guys addressing data set only includes subdivisions that are required for addressing.
+
+#### How do I add or modify subdivisions for a country?
+You can modify the subdivision data returned for a specific country by subscribing to the `AddressEvents::SUBDIVISIONS` event. The Address module includes an example *subdivisions* event subscriber in its test code. See `address/tests/modules/address_test/src/EventSubscriber/GreatBritainEventSubscriber.php`. In this example code, a county field and a predefined list of counties are added for the United Kingdom (country code *GB*).
+
+When *United Kingdom* is selected as the country, a *County* select field appears with the list of options we've defined:
+
+![Custom subdivisions example](../../images/address-countries-4.png)
+
+To start, we subscribe to the `AddressEvents::SUBDIVISIONS` event and implement its callback method: `onSubdivisions()`. We only include the Welsh countries here, for simplicity; a real subscriber would include the full list, sourced from the [CLDR "Territory Subdivisions" listing].
```

Comment:
s/countries/counties

Reply:
Think this should actually be counties, not countries.